Clinical-Genomics/scout
# Move Clinvar submissions code at the institute level and rename submission samples

Looks like it's a lot of changes but it's because I moved the code for the clinvar submissions from cases blueprint to institute blueprint. Clinvar submission are done and classified at the instutute level so it's more correct this way.

New feature and fix #1854 to rename individuals in casedata using the form:

![image](https://user-images.githubusercontent.com/28093618/84258486-4d6fee00-ab17-11ea-8e69-9f751aa25116.png)


When the user renames a sample all samples with that name will be renamed in the form and in casedata downloaded file

**How to test**:
1. Install locally or on stage. 
1. Assign a phenptype (OMIM, HPO) to a case
1. Pin one or more of its variants
1. Go to the page of one of its pinned variants and click to Submit to clinvar
1. Follow the instructions and save the submission
1. Go to the institute submissions page (see figure above) and modify the name of the casedata individual.
1. Make sure that the downloadable casedata file contains the updated individual name

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
